caas/application.go

```diff
@@ -179,3 +179,10 @@ const (
 	RunAsSudoer  RunAs = "sudoer"
 	RunAsNonRoot RunAs = "non-root"
 )
+
+// CharmContainerResourceRequirements defines the memory resource constraints
+// for the workload pod's charm container.
+type CharmContainerResourceRequirements struct {
+	MemRequest uint64
```

Comment:
Can you make it clear here what the unit is for this? i.e what is expected. Is it the number of bytes or MiB?

Reply:
Comment applies for both members of the struct.